packages/examples/src/relay/Feedback/Feedback.tsx

```diff
@@ -1,4 +1,4 @@
-import { graphql, useFragment, useMutation } from "@nova/react";
+import { graphql, useFormat, useFragment, useMutation } from "@nova/react";
```

Comment:
Did we change that component or just added unused import? I would be good to keep pure-relay and relay in sync

Reply:
Ah good catch, it's just left over import from when I was trying something.

Reply:
@sjwilczynski the problem with the `relay` one is that it uses Graphitation payload generator which doesn't support  client side fields (yet).

Only the `pure-relay` example has viewData today.